### PR TITLE
First test with Github Pages

### PR DESCRIPTION
**Pull request by anhiga**

### CODE REVIEW

**Comment by Juanlu001** [comment]
Reopening to trigger Travis (hope it works!)

**Comment by Juanlu001** [comment]
https://travis-ci.org/poliastro/poliastro.github.io-source

**Comment by Juanlu001** [comment]
In this build deployment failed:

https://travis-ci.org/poliastro/poliastro.github.io-source/builds/253625176

But this one succeeded:

https://travis-ci.org/anhiga/anhiga.github.io-source/jobs/253622321

I guess it's the mechanism of the pull request.

The question is: will a direct push to `master` in this repository succeed? We will see after merging this :)

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+
+  # Update conda and create test environment
+  - conda config --set always_yes yes --set changeps1 no
+  - conda env create -q -f environment.yml -n test-environment
+  - source activate test-environment
+install: true
+before_script:
+  - git config --global user.email "poliastro@poliastro.com"
```
Comment (Juanlu001) [inline .travis.yml]: Is it mandatory to have an email address? I prefer leaving this empty than having dummy values. Try removing these two config values, and if there are problems let's just leave it this way.

File: pelicanconf.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 
-AUTHOR = 'Pybonacci'
-SITENAME = 'Pybonacci'
-SITESUBTITLE = u'Blog sobre Python científico en español'
+AUTHOR = 'Poliastro'
+SITENAME = 'Poliastro'
+SITESUBTITLE = u'Blog sobre Poliastro'
```
Comment (Juanlu001) [inline pelicanconf.py]: "poliastro website"

And by the way, use it in small caps everywhere. It's a silly thing but... let's be consistent with capitalization

File: pelicanconf.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 
-AUTHOR = 'Pybonacci'
-SITENAME = 'Pybonacci'
-SITESUBTITLE = u'Blog sobre Python científico en español'
+AUTHOR = 'Poliastro'
```
Comment (Juanlu001) [inline pelicanconf.py]: "The poliastro development team"

File: pelicanconf.py [hunk 4]
```diff
@@ -59,9 +59,9 @@
 # THEME SETTINGS
 THEME = './theme/'
 
-ABOUT_PAGE = '/pages/acerca-de-pybonacci.html'
-TWITTER_USERNAME = 'Pybonacci'
-GITHUB_USERNAME = 'Pybonacci'
+ABOUT_PAGE = '/pages/about-poliastro.html'
+#TWITTER_USERNAME = 'Poliastro'
+GITHUB_USERNAME = 'Poliastro'
```
Comment (Juanlu001) [inline pelicanconf.py]: small caps again

File: pelicanconf.py [hunk 5]
```diff
@@ -73,5 +73,5 @@
 
 # Footer info
 
-LICENSE_URL = "https://github.com/Juanlu001/pybonacci.github.io-source/blob/master/LICENSE"
+LICENSE_URL = "https://github.com/poliastro/poliastro.github.io-source/blob/master/LICENSE"
 LICENSE = "MIT"
```
Comment (Juanlu001) [inline pelicanconf.py]: Let's do "CC-BY for content and MIT for code"

File: publishconf.py [hunk 6]
```diff
@@ -17,4 +17,4 @@
 
 DELETE_OUTPUT_DIRECTORY = True
 
-ISSO_HOST = 'https://comments.pybonacci.es'
+#ISSO_HOST = 'https://comments.pybonacci.es'
```
Comment (Juanlu001) [inline publishconf.py]: Leave it commented but change the host to 'https://comments.example.com', I'd prefer not to have Pybonacci references here. Just for cleanliness :)

File: theme/templates/base.html [hunk 7]
```diff
@@ -56,7 +56,7 @@
 		    <span class="icon-bar"></span>
 		    <span class="icon-bar"></span> 
 		  </button>
-                  <a class="navbar-brand" href="/" title="Home" class="title">Pybonacci</a><!-- — Python y Ciencia-->
+                  <a class="navbar-brand" href="/" title="Home" class="title">Poliastro</a><!-- — Python y Ciencia-->
```
Comment (Juanlu001) [inline theme/templates/base.html]: small caps, and remove the commented subtitle

File: theme/templates/main.css [hunk 8]
```diff
@@ -0,0 +1,330 @@
+body {
```
Comment (Juanlu001) [inline theme/templates/main.css]: Let's keep this off the repository. We might want to change it - I guess it implies another build step? Your take :)

**Comment by anhiga** [comment]
* Tried to fix all the errors. Regarding the main.css file, I would rather keep it as is. Otherwise, `make publish` didn't work by itself.
* Encrypted my Github Token with public key from poliastro.github.io-source repo in Travis CI, hopefully the deployment would work.

**Comment by Juanlu001** [comment]
Then I'm merging as it is! Feel free to push directly to master to make it work - we can clean the history later.

**Pull request merged**